src/main.c

```diff
@@ -85,6 +87,7 @@ int get_key(void) {
 void press_exit(void) {
 	printf("Press any key to exit this application.\n");
 	get_key();
+	sceKernelExitProcess(0);
```

Comment:
from what I can see exit() should just call sceKernelExitProcess https://github.com/vitasdk/newlib/blob/277257a4eee8ae7dcee9fedc3283d948c6d1f6b6/newlib/libc/sys/vita/syscalls.c#L40